### PR TITLE
Initialize create action in Calendars controller

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR adds the createAction method in CalendarController, with the associated routing and twig file.

# Pull Request type

This is a new feature

# How to test

You can go to http://{your_nmm_url}/mtt/networks/network:BIBUS/calendars/create and you will see the title `Création d'un calendrier`.

# Team reviewers 
@datanel @dvdn @djerrah @nberard

### CODE REVIEW

File: Controller/DefaultController.php [hunk 1]
```diff
@@ -6,23 +6,6 @@
 
 class DefaultController extends AbstractController
 {
-    private function findNetwork($externalNetworkId, $networks)
```
Comment (datanel) [inline Controller/DefaultController.php]: it seems not used? do you confirm? @tchevily @vchabot

File: Controller/CalendarController.php [hunk 2]
```diff
@@ -7,6 +7,17 @@
  */
 class CalendarController extends AbstractController
 {
+
+    public function createAction($externalNetworkId)
```
Comment (datanel) [inline Controller/CalendarController.php]: a calendar is not attached to a network
we don't this parameter in the route

File: Resources/config/routing.yml [hunk 3]
```diff
@@ -122,6 +122,10 @@ canal_tp_mtt_block_delete:
         externalNetworkId: -?[a-zA-Z0-9_\-:]*
 
 #Calendar
+canal_tp_mtt_calendar_create:
+    pattern: /networks/{externalNetworkId}/calendars/create
```
Comment (datanel) [inline Resources/config/routing.yml]: you can remove externalNetworkId as a calendar is available for all the networks

File: Tests/Functional/Controller/AbstractControllerTest.php [hunk 4]
```diff
@@ -23,7 +23,7 @@ private function mockDb()
         $this->reloadMttFixtures();
     }
 
-    protected function logIn()
+    protected function logIn($userName, $userPass, $email, array $roles = array(), $sessionAppKey = 'sam_selected_application', $sessionAppValue = 'sam')
```
Comment (datanel) [inline Tests/Functional/Controller/AbstractControllerTest.php]: the parameters are not used

File: Resources/config/routing.yml [hunk 5]
```diff
@@ -122,6 +122,10 @@ canal_tp_mtt_block_delete:
         externalNetworkId: -?[a-zA-Z0-9_\-:]*
 
 #Calendar
+canal_tp_mtt_calendar_create:
+    pattern: /calendars/create
```
Comment (datanel) [inline Resources/config/routing.yml]: pattern is deprecated, use path instead

https://github.com/symfony/symfony/pull/6738/files#diff-0

**Comment by tchevily** [comment]
Jenkins can you test this please?

**Pull request merged**